lambda/delete-email-subscriptions/delete-email-subscriptions.ts

```diff
@@ -50,7 +50,9 @@ export const deleteEmailSubscription = async (userData: UserData) => {
   const config = getRequestConfig(GOV_ACCOUNTS_PUBLISHING_API_TOKEN);
 
   const response = await fetch(deleteUrl, config);
-  if (!response.ok) {
+  if (response.status === 404) {
```

Comment:
I think it would be helpful to have a comment explaining WHY a 404 is treated differently to any other non 200 status

Reply:
Done thanks